flight_controller_2.py

```diff
@@ -36,40 +39,67 @@ def main():
         '''
 
         #Variables.
-        Vref = 5.46
+        Vref            = 5.46
+        CLK             = 16
+        Dout            = 15
+        Din             = 13
+        CS              = 11
+
 
         #Sensors.
+        camera          = Camera('Camera')
+        gps             = GPS('GPS')
+        thermocouple    = AnalogSensor('Outside_temp', Vref, CLK, Dout, Din, CS, [0,0,0], -250, 200)
 
         #Queue.
-        sensors = []
```

Comment:
I changed the name of the list of sensors, because it got a bit confusing later:
          except:
                sensors.remove(sensor)
I thought this just cleared things up a bit.